autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/UpdateFailedHandler.java

```diff
@@ -73,6 +79,7 @@ public void onApplicationEvent(UpdateFailedEvent event) {
             }
             suspendCluster(cluster);
             updateFailures.remove(autoscaleClusterId);
+            scalingTriggerService.create(cluster, TriggerStatus.METRICS_COLLECTION_FAILED, null, Instant.now().toEpochMilli());
```

Comment:
See the point about re-purposing the YARN field for generic information (likely structured), and include information related to the error in this field.